### DISCUSSION TITLE
sdxl-turbo model settings

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/Fooocus#1598] [Q&A] [answered]
Hi, what is the recommended setting for SDXL Turbo models, Thank You

**Comment by xhoxye** [comment, marked as answer]
Please follow the instructions of the model author to set it up

**Comment by siriume** [comment]
> 您好，SDXL Turbo型号的推荐设置是什么，谢谢

check this: #1580

**Reply by mashb1t** [reply]
@wisdombox44  there also is an extensive thread in https://github.com/lllyasviel/Fooocus/discussions/1060 which provides various insights and examples

**Comment by ItsNoted** [comment]
I show how to do it using DreamShaperXL Turbo here. It's much easier than you think!
https://www.youtube.com/watch?v=S4b5tQoUfBA
